src/editor-sidebar/create-panel.js

```diff
@@ -34,19 +34,22 @@ export const CreateThemePanel = ( { createType } ) => {
 		subfolder: '',
 	} );
 
-	useSelect( ( select ) => {
-		const themeData = select( 'core' ).getCurrentTheme();
-		setTheme( {
-			...theme,
-			subfolder:
-				themeData.stylesheet.lastIndexOf( '/' ) > 1
-					? themeData.stylesheet.substring(
-							0,
-							themeData.stylesheet.lastIndexOf( '/' )
-					  )
-					: '',
-		} );
-	}, [] );
+	useSelect(
```

Comment:
This is a lint autofix.

Reply:
This is the only change to create-panel but now I'm seeing an error thrown.

Reply:
I rolled it back to what's in trunk and the error is gone.

Reply:
I rolled this back so that the branch was in a working state... 
This change broke the app and I'm not sure why.  I also attempted to remove the array rather than add `theme` to it with the same problem.

I'm not sure how to proceed but my 'druthers is to leave the warning for now.